src/headless/DisplayInfoHeadless.cpp

```diff
@@ -0,0 +1,22 @@
+#include "DisplayInfo.h"
+#include "UserInteractions.h"
+
+namespace Surge
+{
+namespace GUI
+{
+
+using namespace VSTGUI;
+    
+float getDisplayBackingScaleFactor(CFrame *)
+{
+    return 1.0;
+}
+    
+CRect getScreenDimensions(CFrame *)
+{
+    return CRect(CPoint(0,0), CPoint(1024,768));
```

Comment:
Why this can't just give zero rect? If there is a reason to give non-zero, please comment it.

Reply:
because I just mindlessly copied the linux one which was also not implemented! Good idea.